challenge-manager/challenge-tree/generalized_path_timer_test.go

```diff
@@ -329,13 +361,13 @@ func TestHasConfirmableAncestor(t *testing.T) {
 func TestComputeHonestPathTimer(t *testing.T) {
 	edges := buildEdges(
 		// Alice.
-		newEdge(&newCfg{t: t, edgeId: "blk-0.a-16.a", createdAt: 1}),
+		newEdge(&newCfg{t: t, edgeId: "blk-0.a-16.a", createdAt: 1, claimId: "ass.a"}),
```

Comment:
Is it supposed to represent "associated" ? 😅

Reply:
"assertion" :rofl: